### ISSUE TITLE
Ads

### ISSUE THREAD

**Issue by Eramik** [juliandescottes/piskel#504]
You should add ads. Even if you don't need it, you should add the button "View ad - help the project". So that's will be not annoying and people who want help will use it.
On earned money you will be able to do good things.

**Comment by epicabsol** [comment]
I agree, although I think ads should be a checkbox (off by default),
because passively loading ads every time the site is visited will be much
more effective than looking at one or two manually, requiring users to
suspend working while they view the ad. I'd gladly turn on ads to support
the project, as long as the ads didn't get in the way of the user
interface. However, I don't see much empty space in the editor for placing
ads. Maybe at the bottom of the save or import or export panes?

Ben The Great is in the house!

On Fri, Jul 15, 2016 at 10:34 AM, Denis Godovanets <notifications@github.com

**Comment by Eramik** [comment]
> Maybe at the bottom of the save or import or export panes?
Yes, it is a good place.
>  I think ads should be a checkbox (off by default)
Agree

**Comment by Cruors** [comment]
+1

**Comment by juliandescottes** [comment]
Thanks for this topic! 

I hadn't responded so far but I'm actually experimenting with ads right now.  I should write something in more details, but hosting piskelapp.com costs ~100 dollars per month right now and I want to find a way to cover those costs. I'm not a big fan of ads, but that's one way to do it.

So right now there is a single ad on the bottom of the home page. Given the placement I think nobody should be bothered by it. I'm not expecting much from this but it's mostly for testing (see what kind of content gets displayed, see if I receive hate mail about this etc...)

If there's no huge backlash, my current plans are to redo the home page to be more inviting, to have more content (links to profile pages? guides? faqs?) and place a few ads on those pages.

I absolutely want to _avoid_ putting any ad on the editor. I think the editor should be a safe place where you can focus on your sprite.

In the long run, if ads can be a viable solution, it also means I could think about spending more time working on Piskel (rather than a few random hours here and there as I've been doing for the past 4 years).

Let's see how this goes!

**Comment by voithos** [comment]
Another option (or, in addition?) might be to just offer a means of donating for those interested in supporting the project - either directly, or through a recurring system like Patreon.

As an example where this seems to work, the open source Godot game engine has some pretty good support on Patreon (https://www.patreon.com/godotengine), allowing the lead developer to work on the project full time.
